### ISSUE TITLE
[css-shapes] Clarify which `<basic-shape>` are valid in `shape-*`

### ISSUE THREAD

**Issue by cdoublev** [w3c/csswg-drafts#9728]
[`shape-outside`](https://drafts.csswg.org/css-shapes-1/#shape-outside-property) (CSS Shapes 1) and [`shape-inside`](https://drafts.csswg.org/css-shapes-2/#shape-inside-property) (CSS Shapes 2) are defined with this prose:

  > **`<basic-shape>`:** The shape is computed based on the values of one of `inset()`, `circle()`, `ellipse()` or `polygon()`.

It is not clear to me if `xywh()`, `rect()`, `path()`, should be invalid.

In SVG 2, `<basic-shape>` is even further restricted in [`shape-inside`](https://svgwg.org/svg2-draft/text.html#TextShapeInside) and [`shape-substract`](https://svgwg.org/svg2-draft/text.html#TextShapeSubtract), but I assume these are superseded by CSS Shapes.

<sup>This was previously reported in the multi-faceted issue <a href="https://github.com/w3c/csswg-drafts/issues/7390">#7390</a>.</sup>

**Comment by smfr** [comment]
I agree that clarification here is needed. I think all Basic Shapes should be allowed for all shape-accepting properties.

**Comment by tabatkins** [comment]
Agreed.

**Comment by astearns** [comment]
Changes made in https://github.com/w3c/csswg-drafts/commit/b53255a86965c14c25bfad679466a20c9f261df7. I’ll keep this issue open until the shapes-1 is published.
